tools/mtmd/mtmd.cpp

```diff
@@ -146,6 +191,21 @@ struct mtmd_context {
             tok_row_end_trail = true; // add trailing end-of-row token
             ov_img_first      = false; // overview image is last
         }
+
+        if (proj == PROJECTOR_TYPE_ULTRAVOX) {
+            // TODO @ngxson : check if model n_mel is 128 or 80
+            w_filters = whisper_precalc_filters::get_128_bins();
+        }
+
+        // warning messages
+        if (proj == PROJECTOR_TYPE_LLAMA4) {
+            LOG_WRN("%s: llama 4 vision is known to have degraded quality:\n"
+                    "    https://github.com/ggml-org/llama.cpp/pull/13282\n", __func__);
+        }
+        if (has_audio) {
+            LOG_WRN("%s: audio input is in experimental stage and may have reduced quality:\n"
+                    "    https://github.com/ggml-org/llama.cpp/pull/13623\n", __func__);
+        }
```

Comment:
The model hallucinates on audio longer than 1 minute and I'm still not sure why (haven't yet had time to try the same audio on `transformers`)

But I think for now putting a small notice here is enough, this is kinda experimental support for now, hopefully we will get [gemma 3n](https://huggingface.co/google/gemma-3n-E2B-it-litert-preview) supported soon